lib/services/servermanagement/generate.cmd

```diff
@@ -0,0 +1,17 @@
+::

```

Comment:
I keep getting burned on this :)

I was originally working off the `master` branch, but at the end of working on the .net sdk, found out that I should have been working off the `autorest` branch. So, spent some time fixing that one, and then saw the node repo was set the same way, so I figured that was the right thing to do there too, and consequently moved over to working off `autorest` in the node-sdk project too.

Which apparently it ain't... 

:disappointed: 

I'l move it over.